### PR TITLE
behn: refactor to use +abet pattern

### LINKED ISSUE #5747: %behn timers sometimes get stuck
%behn timers sometimes get stuck (don't ever fire for a particular agent), and then only fire when ship is ctrl+d'ed and started back up. cc: @dr-frmr @philipcmonk

### PR DESCRIPTION
**Pull request by joemfb**
This PR is built from my validation code for #5857. I still wasn't sure that %behn was handling all state transitions correctly, so I wrote a more explicit, declarative state machine to check its results. Once that I was done, I felt the result was clearer then the previous code.

#5857 has the necessary changes to fix #5747, this is additional cleanup that I think is worthwhile.

### CODE REVIEW

**Review by belisarius222 — APPROVED** [review]
I requested a comment change, but this LGTM.

File: pkg/arvo/sys/vane/behn.hoon [hunk 1]
```diff
@@ -45,159 +45,109 @@
...
+    |=  vax=vase
+    ^+  this
+    %.  [duct %pass /drip/(scot %ud count.drips.state) %b %wait +(now)]
+    %=  emit
+      movs.drips.state   (~(put by movs.drips.state) count.drips.state vax)
+      count.drips.state  +(count.drips.state)
+    ==
   ::
-  ++  vega  [moves state]
   ::  +wake: unix says wake up; process the elapsed timer and set :next-wake
```
Comment (belisarius222) [inline pkg/arvo/sys/vane/behn.hoon:129]: this looks like it no longer sets .next-wake, since that's now done in +abet

**Review by belisarius222 — APPROVED** [review]
LGTM, much cleaner

File: pkg/arvo/sys/vane/behn.hoon (hunk 1 above)
Comment (joemfb) [reply pkg/arvo/sys/vane/behn.hoon]: good catch.

**Pull request merged**